bundler/benchmarks/bm_bundle_install_rails.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'support/benchmark_bundler.rb'
+require 'bundler/cli'
+
+# Fixed Nokogiri version because latest version 1.6.8 fails to install
```

Comment:
Is 1.6.8 failing to install in the Docker image? If so, I'll have to fix that :smile: